web/auth/oauth.go

```diff
@@ -914,6 +914,25 @@ func accessToken(c echo.Context) error {
 	return c.JSON(http.StatusOK, out)
 }
 
+func impersonatedAccessToken(c echo.Context) error {
```

Comment:
I don't like the `impersonate`. It implies that the token will be used by something else that is not the konnector. But, here, the goal is to provide the token to the flagship app, so that the client-side konnector can use it.